app/modules/annotations/resources.py

```diff
@@ -87,7 +88,26 @@ def post(self, args):
             if not encounter:
                 abort(code=HTTPStatus.BAD_REQUEST, message='encounter_guid not found')
             args['encounter_guid'] = encounter_guid
-
+        elif hasattr(asset.git_store, 'get_asset_group_sightings_for_asset'):
```

Comment:
@bluemellophone, this is effectively a way of asking if the git_store is an AssetGroup. 
This change has specifically not done anything WRT MWS. 
Currently MWS can create an encounter less annotation at any point. 
I'd suggest that you may want to thinnk about this ...... under a separate PR ;)